pkg/archive/archive.go

```diff
@@ -54,6 +54,9 @@ type (
 		// replaced with the matching name from this map.
 		RebaseNames map[string]string
 		InUserNS    bool
+		// Use "user.*" xattrs instead of "trusted.*" ones.
+		// Only meaningful for OverlayWhiteoutFormat.
+		UserXAttr bool
```

Comment:
Is there a specific subset of `user.*` attributes that's used for this? (looks like we need a "allow" / "deny" list?